rosdep/python.yaml

```diff
@@ -701,6 +701,14 @@ python-falcon:
   fedora: [python-falcon]
   ubuntu: [python-falcon]
 python-fcn-pip:
+  debian:
+    pip:
+      depends: [libhdf5-dev, liblapack-dev]
+      packages: [fcn]
+  fedora:
+    pip:
+      depends: [hdf5-devel, lapack-devel]
```

Comment:
I see, but why depend names on osx is different from debian @wkentaro 
anyway, fixed to use rosdep key names.